app/tests/unit/components/Form/ProjectMilestoneReportFormSummary.test.tsx

```diff
@@ -141,5 +148,258 @@ describe("The Project Milestone Report Form Summary", () => {
+    expect(screen.getByText(/\$891\.00/i)).toBeInTheDocument();
+    expect(screen.getByText(/\$111\.00/i)).toBeInTheDocument();
+
+    // adjusted values
+    expect(
+      screen.getByText("Gross Payment Amount This Milestone (Adjusted)")
+    ).toBeInTheDocument();
+    expect(screen.getByText(/\$99\.00/i)).toBeInTheDocument();
+
+    expect(screen.getByText(/\$89\.00/i)).toBeInTheDocument();
+    expect(
+      screen.getByText("Net Payment Amount This Milestone (Adjusted)")
+    ).toBeInTheDocument();
+    expect(screen.getByText(/\$11\.00/i)).toBeInTheDocument();
+    expect(screen.getByText(/\$67\.00/i)).toBeInTheDocument();
+    expect(
+      screen.getByText("Holdback Amount This Milestone (Adjusted)")
+    ).toBeInTheDocument();
+    expect(screen.getByText(/\$88\.00/i)).toBeInTheDocument();
+    expect(screen.getByText(/\$91\.00/i)).toBeInTheDocument();
+  });
+});
+
+const mockQueryPayloadWithDiffs = {
+  Form() {
+    return {
+      jsonSchema: milestoneProdSchema,
+    };
+  },
+  ProjectRevision() {
+    const result = {
+      isFirstRevision: false,
+      summaryMilestoneFormChanges: {
+        edges: [
+          {
+            node: {
+              id: "Test Reporting Requirement ID - 1",
+              isPristine: false,
+              newFormData: {
+                totalEligibleExpenses: 1000,
+                description: "charmander",
+                projectId: 1,
+                reportingRequirementIndex: 1,
+                reportType: "General",
+                reportDueDate: "2020-01-10T23:59:59.999-07:00",
+                reportingRequirementId: 1,
+                hasExpenses: true,
+                calculatedNetAmount: 221,
+                calculatedGrossAmount: 222,
+                calculatedHoldbackAmount: 223,
+                adjustedNetAmount: 21,
+                adjustedGrossAmount: 22,
+                adjustedHoldBackAmount: 23,
+              },
+              operation: "UPDATE",
+              formChangeByPreviousFormChangeId: {
+                newFormData: {
+                  description: "bulbasaur",
+                  projectId: 1,
+                  reportingRequirementIndex: 1,
+                  reportDueDate: "2020-01-01T13:59:59.999-07:00",
+                  reportType: "Advanced",
+                  reportingRequirementId: 1,
+                  hasExpenses: true,
+                  calculatedNetAmount: 111,
+                  calculatedGrossAmount: 112,
+                  calculatedHoldbackAmount: 113,
+                  adjustedNetAmount: 11,
+                  adjustedGrossAmount: 12,
+                  adjustedHoldBackAmount: 13,
+                },
+              },
+              formDataRecordId: 1,
+            },
+          },
+        ],
+      },
+    };
+    return result;
+  },
+};
+
+describe("Project Milestone Summary with diffs, without latestCommittedData", () => {
+  it("Displays diffs of the data fields that were updated and the old values", () => {
```

Comment:
This seems to be testing the same thing as `it("Displays diffs of the the data fields that were updated"`?  Do we need this test?

Reply:
I just added it for more coverage, the first one shows "Not Entered" this one shows actual diffs.  Happy to remove it.